.github/workflows/validate-yaml-syntax.yaml

```diff
@@ -0,0 +1,15 @@
+name: validate-yaml-syntax
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  validate-yaml:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Validate YAML file
+        run: yamllint -c .yamllint-config linkml_model/model/schema/*.yaml
```

Comment:
How does `yamllint` get installed in this workflow?

Reply:
yamllint is already installed on Ubuntu-based GitHub runners.

Reply:
Whoa! https://github.com/actions/runner-images/issues/1142 So it is. The more you know!